weighted_circumcenter_translateC2<RT>

https://github.com/CGAL/cgal/blob/master/Cartesian_kernel/include/CGAL/constructions/kernel_ftC2.h#L458

there is a call `weighted_circumcenter_translateC2(qx-px, qy-py, dqw,rx-px, ry-py,drw,x, y);`. For number types using expression templates, we need to specify the template parameter with `weighted_circumcenter_translateC2<RT>(qx-px, qy-py, qw-pw, rx-px, ry-py, rw-pw, x, y);`

Is there any platform in the testsuite that has GMPXX?

And there is the compilation error with GMPXX:
```
[ 50%] Building CXX object CMakeFiles/test_regular_triangulation_2.dir/test_regular_triangulation_2.cpp.o
/usr/bin/c++  -DCGAL_TEST_SUITE -DCGAL_USE_GMP -DCGAL_USE_GMPXX -DCGAL_USE_MPFR -I/home/cgal_tester/build/src/cmake/platforms/Fedora-Release/test/Triangulation_2/include -I/home/cgal_tester/build/src/cmake/platforms/Fedora-Release/test/Triangulation_2/../../include -I/home/cgal_tester/build/src/cmake/platforms/Fedora-Release/test/Triangulation_2 -I/home/cgal_tester/build/src/cmake/platforms/Fedora-Release/include -I/mnt/testsuite/include  -Wall -frounding-math -O3 -DCGAL_NDEBUG   -o CMakeFiles/test_regular_triangulation_2.dir/test_regular_triangulation_2.cpp.o -c /home/cgal_tester/build/src/cmake/platforms/Fedora-Release/test/Triangulation_2/test_regular_triangulation_2.cpp
In file included from /mnt/testsuite/include/CGAL/predicates/kernel_ftC2.h:32:0,
                 from /mnt/testsuite/include/CGAL/Cartesian/Line_2.h:30,
                 from /mnt/testsuite/include/CGAL/Cartesian/Cartesian_base.h:36,
                 from /mnt/testsuite/include/CGAL/Simple_cartesian.h:28,
                 from /home/cgal_tester/build/src/cmake/platforms/Fedora-Release/test/Triangulation_2/include/CGAL/_test_types.h:36,
                 from /home/cgal_tester/build/src/cmake/platforms/Fedora-Release/test/Triangulation_2/test_regular_triangulation_2.cpp:24:
/mnt/testsuite/include/CGAL/constructions/kernel_ftC2.h: In instantiation of 'void CGAL::weighted_circumcenterC2(const RT&, const RT&, const We&, const RT&, const RT&, const We&, const RT&, const RT&, const We&, RT&, RT&) [with RT = __gmp_expr<__mpq_struct [1], __mpq_struct [1]>; We = __gmp_expr<__mpq_struct [1], __mpq_struct [1]>]':
/mnt/testsuite/include/CGAL/Kernel/function_objects.h:3715:28:   required from 'CGAL::CommonKernelFunctors::Construct_weighted_circumcenter_2<K>::result_type CGAL::CommonKernelFunctors::Construct_weighted_circumcenter_2<K>::operator()(const Weighted_point_2&, const Weighted_point_2&, const Weighted_point_2&) const [with K = CGAL::Simple_cartesian<__gmp_expr<__mpq_struct [1], __mpq_struct [1]> >; CGAL::CommonKernelFunctors::Construct_weighted_circumcenter_2<K>::result_type = CGAL::Point_2<CGAL::Simple_cartesian<__gmp_expr<__mpq_struct [1], __mpq_struct [1]> > >; CGAL::CommonKernelFunctors::Construct_weighted_circumcenter_2<K>::Weighted_point_2 = CGAL::Weighted_point_2<CGAL::Simple_cartesian<__gmp_expr<__mpq_struct [1], __mpq_struct [1]> > >]'
/mnt/testsuite/include/CGAL/Lazy.h:1672:3:   required from 'CGAL::Lazy_construction<LK, AC, EC, E2A_, true>::result_type CGAL::Lazy_construction<LK, AC, EC, E2A_, true>::operator()(const L0&, const L1&, const L2&) const [with L0 = CGAL::Weighted_point_2<CGAL::Epeck>; L1 = CGAL::Weighted_point_2<CGAL::Epeck>; L2 = CGAL::Weighted_point_2<CGAL::Epeck>; LK = CGAL::Epeck; AC = CGAL::CommonKernelFunctors::Construct_weighted_circumcenter_2<CGAL::Simple_cartesian<CGAL::Interval_nt<false> > >; EC = CGAL::CommonKernelFunctors::Construct_weighted_circumcenter_2<CGAL::Simple_cartesian<__gmp_expr<__mpq_struct [1], __mpq_struct [1]> > >; E2A_ = CGAL::Default; CGAL::Lazy_construction<LK, AC, EC, E2A_, true>::result_type = CGAL::Point_2<CGAL::Epeck>]'
/mnt/testsuite/include/CGAL/Regular_triangulation_2.h:1169:66:   required from 'CGAL::Regular_triangulation_2<Gt, Tds>::Bare_point CGAL::Regular_triangulation_2<Gt, Tds>::weighted_circumcenter(const Weighted_point&, const Weighted_point&, const Weighted_point&) const [with Gt = CGAL::Epeck; Tds = CGAL::Triangulation_data_structure_2<CGAL::Regular_triangulation_vertex_base_2<CGAL::Epeck, CGAL::Triangulation_ds_vertex_base_2<void> >, CGAL::Regular_triangulation_face_base_2<CGAL::Epeck, CGAL::Triangulation_face_base_2<CGAL::Epeck, CGAL::Triangulation_ds_face_base_2<void> > > >; CGAL::Regular_triangulation_2<Gt, Tds>::Bare_point = CGAL::Point_2<CGAL::Epeck>; CGAL::Regular_triangulation_2<Gt, Tds>::Weighted_point = CGAL::Weighted_point_2<CGAL::Epeck>]'
/home/cgal_tester/build/src/cmake/platforms/Fedora-Release/test/Triangulation_2/test_regular_triangulation_2.cpp:35:22:   required from here
/mnt/testsuite/include/CGAL/constructions/kernel_ftC2.h:458:36: error: no matching function for call to 'weighted_circumcenter_translateC2(__gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_binary_minus> >, __gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_binary_minus> >, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>&, __gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_binary_minus> >, __gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_binary_minus> >, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>&, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>&, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>&)'
   weighted_circumcenter_translateC2(qx-px, qy-py, dqw,rx-px, ry-py,drw,x, y);
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/testsuite/include/CGAL/constructions/kernel_ftC2.h:423:1: note: candidate: template<class RT> void CGAL::weighted_circumcenter_translateC2(const RT&, const RT&, const RT&, const RT&, const RT&, const RT&, RT&, RT&)
 weighted_circumcenter_translateC2(const RT &dqx, const RT &dqy, const RT &dqw,
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/testsuite/include/CGAL/constructions/kernel_ftC2.h:423:1: note:   template argument deduction/substitution failed:
/mnt/testsuite/include/CGAL/constructions/kernel_ftC2.h:458:36: note:   deduced conflicting types for parameter 'const RT' ('__gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_binary_minus> >' and '__gmp_expr<__mpq_struct [1], __mpq_struct [1]>')
   weighted_circumcenter_translateC2(qx-px, qy-py, dqw,rx-px, ry-py,drw,x, y);
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
``

https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.11-Ic-61/Triangulation_2/TestReport_lrineau_Fedora-Release.gz

--- a/Cartesian_kernel/include/CGAL/constructions/kernel_ftC2.h
+++ b/Cartesian_kernel/include/CGAL/constructions/kernel_ftC2.h
@@ -455,7 +455,7 @@ weighted_circumcenterC2( const RT &px, const RT &py, const We &pw,
   RT dqw = RT(qw-pw);
   RT drw = RT(rw-pw);
 
-  weighted_circumcenter_translateC2(qx-px, qy-py, dqw,rx-px, ry-py,drw,x, y);
+  weighted_circumcenter_translateC2<RT>(qx-px, qy-py, dqw,rx-px, ry-py,drw,x, y);
   x += px;
   y += py;
 }

